ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -64,6 +65,11 @@ public void setBorderStyle(ReactViewGroup view, @Nullable String borderStyle) {
     view.setBorderStyle(borderStyle);
   }
 
+  @ReactProp(name = "elevation")
+  public void setElevation(ReactViewGroup view, float elevation) {
+    ViewCompat.setElevation(view, PixelUtil.toPixelFromDIP(elevation));
```

Comment:
Why not check for platform level and call `View.setElevation`? http://developer.android.com/reference/android/view/View.html#setElevation(float)?